### PR TITLE
Add max-retry option for rabbitMQ consumer

### PR DESCRIPTION
**Pull request by TomPradat**

### CODE REVIEW

**Comment by TomPradat** [comment]
Ping @Th3Mouk

**Comment by TomPradat** [comment]
Can you merge it since you approved the changes ?

File: src/Router/RabbitMq/Adapter.php [hunk 1]
```diff
@@ -52,58 +76,75 @@ public function __invoke(Message $message)
 
 //        $subjectObs->subscribe($this->logger);
 
-        $subjectObs
+        if (null !== $this->timeout) {
             // Give only x ms to execute
-            ->timeout($this->timeout)
+            $subjectObs->timeout($this->timeout);
```
Comment (Th3Mouk) [inline src/Router/RabbitMq/Adapter.php]: `$subjectObs = $subjectObs->timeout($this->timeout);`

Comment (TomPradat) [reply src/Router/RabbitMq/Adapter.php]: Alright 👍

**Pull request merged**